developer-docs-site/docusaurus.config.js

```diff
@@ -12,8 +12,8 @@ const config = {
   tagline: "Developer Documentation",
   url: "https://aptos.dev",
   baseUrl: "/",
-  onBrokenLinks: "warn",
-  onBrokenMarkdownLinks: "warn",
+  onBrokenLinks: "throw",
+  onBrokenMarkdownLinks: "throw",
```

Comment:
We don't have any external links do we?  Because then it could fail erratically

Reply:
I think we should still know. Either that or get an alarm somehow when it throws a warning. Also this is `throw` level by default, I figure they must be doing retries and stuff. I can confirm that later though.